PendingReleaseNotes

```diff
@@ -496,3 +496,11 @@ Relevant tracker: https://tracker.ceph.com/issues/55521
   (``auth_client_required=cephx`` and ``ms_mon_client_mode=secure``).
   If you have cephx authentication disabled on your cluster, you may
   need to adjust these settings for RGW.
+* telemetry: Added new Rook metrics to the 'basic' channel to report Rook's
```

Comment:
Looks good; just needs a newline after "need to adjust these settings for RGW".

Reply:
thanks, fixed!